python/lsst/analysis/tools/actions/vector/mathActions.py

```diff
@@ -0,0 +1,132 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = (
+    "ConstantValue",
+    "AddVector",
+    "SubtractVector",
+    "DivideVector",
+    "MultiplyVector",
+    "FractionalDifference",
+)
+
+import logging
+
+import numpy as np
+from lsst.pex.config import Field
+from lsst.pex.config.configurableActions import ConfigurableActionField
+
+from ...interfaces import KeyedData, KeyedDataSchema, Vector, VectorAction
+
+_LOG = logging.getLogger(__name__)
+
+
+class ConstantValue(VectorAction):
```

Comment:
Most vector actions in the `mathActions` file end with the suffix `Vector` (all except here and `FractionalDifference`). Is it worth considering making that consistent here too? E.g., `ConstantVector`? `ConstantValueVector`?

Reply:
I was actually considering it renaming it to `Scalar`something rather than `Constant`something, but decided against it.

But to the extent that there is a naming scheme, the `Vector`-suffixed actions all begin with verbs (see also `LoadVector` and `DownselectVector` in `vectorActions.py`, whereas the unsuffixed actions are nouns (`ConstantValue`, `MagDiff` as short for `MagnitudeDifference`, etc). In principle you could verbify the various `Diff(erence)` actions since difference is a verb too, but I don't think the names would be any clearer. For `ConstantValue`, I can't think of a good way to verbify it at all.

Reply:
Fair enough. In which case, I suggest leaving this (and the other non-conformant action here) as-is for the time being. Further changes can be punted to a future ticket, if desired.